src/ethereum/index.ts

```diff
@@ -450,39 +482,41 @@ export class Ethereum extends Blockchain {
    * Stakes funds into pool.
    *
    * @param address - Sender address.
-   * @param amount - Stake amount in ETH.
+   * @param amount - Stake amount in wei as a BigNumber.
    * @param source - Stake source. Default is '0'.
    *
    * @returns A Promise that resolves to the unsigned ETH transaction object.
    *
-   * @throws Will throw an Error if the amount is not a string, the amount is less than the minimum, or the contract call fails.
+   * @throws Will throw an Error if the amount is not a BigNumber, the amount is less than the minimum, or the contract call fails.
    */
 
   public async stake(
     address: string,
-    amount: string,
+    amount: BigNumber,
```

Comment:
Okay, if we change the amount from string to BigNumber, we have to do it in every method of the Ethereum module (e.g., unstake, simulateUnstake, unstakePending, and so on). 

Also, since BigNumber is a separate library, I think we need to make it easier for users to use it. Let's reexport this lib

```
import BigNumber from 'bignumber.js';
export { BigNumber };
```
and it could be possible to use it like this `import { BigNumber } from '@everstake/wallet-sdk';`

But I would keep it as String for simplicity. @Messer4 wdyt?